public/examples/google-pay.html

```diff
@@ -36,10 +36,11 @@
         return googlePay;
       }
 
-      async function createPayment(token) {
+      async function createPayment(token, verificationToken) {
```

Comment:
Alternatively I can make a separate google-pay-with-verification example but I figured it was more clear to use the existing one. Open to feedback.

Reply:
ended up going with this